Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Xamarin.Forms.Controls.Issues.Shared.projitems

```diff
@@ -2491,6 +2494,7 @@
     </EmbeddedResource>
   </ItemGroup>
   <ItemGroup>
+    <EmbeddedResource Include="$(MSBuildThisFileDirectory)Issue6742.xaml">
```

Comment:
There is a small error here:

`Error MSB4024: The 'EmbeddedResource' start tag on line 2497 position 6 does not match the end tag of 'ItemGroup'. Line 2506, position 5. (MSB4024) (Xamarin.Forms.ControlGallery.iOS)`

The build is failing for this reason, could you review it? Let me know if you need help with something.

Reply:
So sorry - fixed.